### ISSUE TITLE
absent: ext-*axis' Renderer inner property

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1806]
Found: Ext.NET 7.0.0-preview4_2020-07-21.

While this looks similar to #1738, it is not clear how the renderer should be used when an inline handler is preferred instead of a reference to a JavaScript function (`Fn="x"`), as used in the below two examples:

```
<ext-numericaxis
    fields="IE"
    position="Left"
    grid="true"
    minimum="0">
    <renderer handler="return label.toFixed(label < 10 ? 1: 0) + '%';" />
</ext-numericaxis>
<ext-categoryaxis position="Bottom" fields="Date" grid="true">
    <label rotationdegrees="-90" />
    <renderer handler="return Ext.Date.format(label, 'M');" />
</ext-categoryaxis>
```

What should valid Ext.NET 7 syntax for that?

#### WebForms examples matching `ext:(Numeric|Category)Axis` and `Renderer`

1. [Chart > Area > Basic](https://examples5.ext.net/#/Chart/Area/Basic/)
2. [Chart > Area > Stacked](https://examples5.ext.net/#/Chart/Area/Stacked/)
3. [Chart > Area > Stacked_100](https://examples5.ext.net/#/Chart/Area/Stacked_100/)
4. [Chart > Bar > 3D](https://examples5.ext.net/#/Chart/Bar/3D/)
5. [Chart > Bar > Basic](https://examples5.ext.net/#/Chart/Bar/Basic/)
6. [Chart > Bar > Grouped](https://examples5.ext.net/#/Chart/Bar/Grouped/)
7. [Chart > Bar > Renderer](https://examples5.ext.net/#/Chart/Bar/Renderer/)
8. [Chart > Bar > Stacked](https://examples5.ext.net/#/Chart/Bar/Stacked/)
9. [Chart > Bar > Stacked_100](https://examples5.ext.net/#/Chart/Bar/Stacked_100/)
10. [Chart > Column > Basic](https://examples5.ext.net/#/Chart/Column/Basic/)
11. [Chart > Column > Gradients](https://examples5.ext.net/#/Chart/Column/Gradients/)
12. [Chart > Column > Limit](https://examples5.ext.net/#/Chart/Column/Limit/)
13. [Chart > Column > Renderer](https://examples5.ext.net/#/Chart/Column/Renderer/)
14. [Chart > Column > Stacked](https://examples5.ext.net/#/Chart/Column/Stacked/)
15. [Chart > Column > Stacked_100](https://examples5.ext.net/#/Chart/Column/Stacked_100/)
16. [Chart > Combination > Binding_Tabs](https://examples5.ext.net/#/Chart/Combination/Binding_Tabs/)
17. [Chart > Combination > Custom_Theme](https://examples5.ext.net/#/Chart/Combination/Custom_Theme/)
18. [Chart > Combination > Dashboard](https://examples5.ext.net/#/Chart/Combination/Dashboard/)
19. [Chart > Combination > Infographic](https://examples5.ext.net/#/Chart/Combination/Infographic/)
20. [Chart > Combination > Pareto](https://examples5.ext.net/#/Chart/Combination/Pareto/)
21. [Chart > Gauge > Basic_2](https://examples5.ext.net/#/Chart/Gauge/Basic_2/)
22. [Chart > Line > Basic](https://examples5.ext.net/#/Chart/Line/Basic/)
23. [Chart > Line > CrossZoom](https://examples5.ext.net/#/Chart/Line/CrossZoom/)
24. [Chart > Line > Marked_Spline](https://examples5.ext.net/#/Chart/Line/Marked_Spline/)
25. [Chart > Line > Markers](https://examples5.ext.net/#/Chart/Line/Markers/)
26. [Chart > Line > Mixed](https://examples5.ext.net/#/Chart/Line/Mixed/)
27. [Chart > Line > Multiple_Axes](https://examples5.ext.net/#/Chart/Line/Multiple_Axes/)
28. [Chart > Line > Renderer](https://examples5.ext.net/#/Chart/Line/Renderer/)
29. [Chart > Line > Spline](https://examples5.ext.net/#/Chart/Line/Spline/)
30. [Chart > Misc > Captions](https://examples5.ext.net/#/Chart/Misc/Captions/)
31. [Chart > Misc > Merge_Two_Charts](https://examples5.ext.net/#/Chart/Misc/Merge_Two_Charts/)
32. [Chart > Misc > Reload](https://examples5.ext.net/#/Chart/Misc/Reload/)
33. [Chart > Misc > Theme](https://examples5.ext.net/#/Chart/Misc/Theme/)
34. [Chart > Misc > ToolTips](https://examples5.ext.net/#/Chart/Misc/ToolTips/)
35. [Chart > Misc > VerticalMarker](https://examples5.ext.net/#/Chart/Misc/VerticalMarker/)
36. [Chart > Radar > Basic](https://examples5.ext.net/#/Chart/Radar/Basic/)
37. [Chart > Radar > Fill](https://examples5.ext.net/#/Chart/Radar/Fill/)
38. [Chart > Radar > Marked](https://examples5.ext.net/#/Chart/Radar/Marked/)
39. [Chart > Radar > MultiAxis](https://examples5.ext.net/#/Chart/Radar/MultiAxis/)
40. [Chart > Scatter > Basic](https://examples5.ext.net/#/Chart/Scatter/Basic/)
41. [Chart > Scatter > Bubble](https://examples5.ext.net/#/Chart/Scatter/Bubble/)
42. [Combination_Samples > Applications > Word_Wrench](https://examples5.ext.net/#/Combination_Samples/Applications/Word_Wrench/)
43. [Desktop > Introduction > Overview](https://examples5.ext.net/#/Desktop/Introduction/Overview/)

#### MVC examples matching `(Numeric|Category)Axis` and `Renderer`

1. [Chart > Area > Basic](https://mvc5.ext.net/#/Chart_Area/Basic/)
2. [Chart > Area > Stacked](https://mvc5.ext.net/#/Chart_Area/Stacked/)
3. [Chart > Area > Stacked_100](https://mvc5.ext.net/#/Chart_Area/Stacked_100/)
4. [Chart > Bar > Basic](https://mvc5.ext.net/#/Chart_Bar/Basic/)
5. [Chart > Bar > Basic_3D](https://mvc5.ext.net/#/Chart_Bar/Basic_3D/)
6. [Chart > Bar > Grouped](https://mvc5.ext.net/#/Chart_Bar/Grouped/)
7. [Chart > Bar > Renderer](https://mvc5.ext.net/#/Chart_Bar/Renderer/)
8. [Chart > Bar > Stacked](https://mvc5.ext.net/#/Chart_Bar/Stacked/)
9. [Chart > Bar > Stacked_100](https://mvc5.ext.net/#/Chart_Bar/Stacked_100/)
10. [Chart > Column > Basic](https://mvc5.ext.net/#/Chart_Column/Basic/)
11. [Chart > Column > Gradients](https://mvc5.ext.net/#/Chart_Column/Gradients/)
12. [Chart > Column > Limit](https://mvc5.ext.net/#/Chart_Column/Limit/)
13. [Chart > Column > Renderer](https://mvc5.ext.net/#/Chart_Column/Renderer/)
14. [Chart > Column > Stacked](https://mvc5.ext.net/#/Chart_Column/Stacked/)
15. [Chart > Column > Stacked_100](https://mvc5.ext.net/#/Chart_Column/Stacked_100/)
16. [Chart > Combination > Binding_Tabs](https://mvc5.ext.net/#/Chart_Combination/Binding_Tabs/)
17. [Chart > Combination > Custom_Theme](https://mvc5.ext.net/#/Chart_Combination/Custom_Theme/)
18. [Chart > Combination > Dashboard](https://mvc5.ext.net/#/Chart_Combination/Dashboard/)
19. [Chart > Misc > Captions](https://mvc5.ext.net/#/Chart_Misc/Captions/)
20. [Chart > Radar > Basic](https://mvc5.ext.net/#/Chart_Radar/Basic/)
21. [Chart > Scatter > Basic](https://mvc5.ext.net/#/Chart_Scatter/Basic/)
22. [Desktop > Overview](https://mvc5.ext.net/#/Desktop/Overview/)

**Comment by AndreyChechel** [comment]
Probably this comment provides an answer: https://github.com/extnet/Ext.NET/issues/1809#issuecomment-662097657

**Comment by fabriciomurta** [comment]
Yes, the same applies. I just pointed both issues to #1738 instead of pointing them to each other. But now the triad is closed :)
